borgmatic/actions/bootstrap.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import os
+import json
+
+import borgmatic.borg.extract
+import borgmatic.borg.rlist
+import borgmatic.config.validate
+import borgmatic.hooks.command
+
+from borgmatic.borg.state import DEFAULT_BORGMATIC_SOURCE_DIRECTORY
+
+logger = logging.getLogger(__name__)
+
+def get_config_paths(bootstrap_arguments, global_arguments, local_borg_version):
+    borgmatic_source_directory = DEFAULT_BORGMATIC_SOURCE_DIRECTORY
+    borgmatic_manifest_path = os.path.expanduser(
+        os.path.join(borgmatic_source_directory, 'bootstrap', 'configs-list.json')
+    )
+    extract_process = borgmatic.borg.extract.extract_archive(
+        global_arguments.dry_run,
+        bootstrap_arguments.repository,
+        borgmatic.borg.rlist.resolve_archive_name(
+            bootstrap_arguments.repository,
+            bootstrap_arguments.archive or 'latest',
+            {},
+            local_borg_version,
+            global_arguments
+        ),
+        [borgmatic_manifest_path],
+        {},
+        {},
```

Comment:
Given that config won't be available, there might be some additional command-line flags we'll eventually need for duplicating configuration options. But we can probably add those as user feature requests come in rather than trying to do it in advance.